zkdb/src/api/apollo/validation.ts

```diff
@@ -32,6 +32,14 @@ export const validateCID = Joi.string()
   .pattern(/^b[a-z,2,3,4,5,6,7]+(|=)+$/i)
   .message('Invalid CID');
 
+export const validateIndex = Joi.string()
+  .pattern(/^[0-9]+$/)
+  .message('Invalid index');
+
+export const validatePoseidonHash = Joi.string()
+  .pattern(/^[0-9]+$/)
```

Comment:
I think the validation for poseidon digest is wrong, i can be in `Uint8Array, hex string, base32` but it can't be `/^[0-9]+$/`

Reply:
Poseidon hash consists only digits in snarky-js

Reply:
Just like we discussed, `base32` could be a better way to encode for communication